### PR TITLE
Signup command and Arangodb

### PR DESCRIPTION
**Pull request by Nabila63Ahmed**

### CODE REVIEW

File: src/main/java/database/ArangoHandler.java [hunk 1]
```diff
@@ -2,103 +2,117 @@
...
-    public String createUserProfile(UserProfile userProfile) throws ArangoDBException {
-        /* Check validity before insertion */
-        if (!isValidUserProfile(userProfile)) {
-            throw new RuntimeException("Invalid user profile");
-        }
+    public String createUser(Object user) {
+        UserProfile userProfile= (UserProfile) user;
 
-        /* Insert user profile */
         DocumentCreateEntity insertedUserProfile = collection.insertDocument(userProfile);
```
Comment (2shraaf) [inline src/main/java/database/ArangoHandler.java]: What happens if the user already exists?, Do we need to handle an exception?

File: src/main/java/database/ArangoHandler.java [hunk 2]
```diff
@@ -2,103 +2,117 @@
...
-    public boolean isValidUserProfile(UserProfile userProfile) {
-        // TODO: Check if valid email
-        // TODO: Check if email does not already exist
-        return true;
+    public Object getUser(String id) {
+        /* Retrieve user profile */
+        return collection.getDocument(id, UserProfile.class);
     }
 
     public static void main(String[] args) {
```
Comment (2shraaf) [inline src/main/java/database/ArangoHandler.java]: We don't need a main method on dev, if you need it later make TODO to delete it at the end.

File: src/main/java/database/ArangoHandler.java (hunk 1 above)
Comment (Nabila63Ahmed) [reply src/main/java/database/ArangoHandler.java]: Handled at signup. however we can only check existence(hopefully)

File: src/main/java/database/ArangoHandler.java (hunk 2 above)
Comment (Nabila63Ahmed) [reply src/main/java/database/ArangoHandler.java]: ok

File: src/main/java/database/ArangoHandler.java (hunk 1 above)
Comment (Nabila63Ahmed) [reply src/main/java/database/ArangoHandler.java]: I can't

**Pull request merged**